### DISCUSSION TITLE
similarities with publicbodies.org and collaboration

### DISCUSSION THREAD

**Discussion by augusto-herrmann** [govdirectory/website#93] [Ideas]
Hi.

I contribute to [publicbodies.org](https://publicbodies.org/), which is a website created by the [Open Knowledge Labs](http://okfnlabs.org/) in 2011 to collect basic information on governmental organizations (public bodies) from around the world and make them easy to browse and reuse.

The project was stale for a while, but we have since restarted working on in early this year. Here is our Github repo:

* [okfn/publicbodies](https://github.com/okfn/publicbodies)

I have already [set up a bot](https://github.com/okfn/publicbodies/pull/105) to collect data from official open data sources for the Brazilian public bodies, which is automatically updated once a week, using Github Actions.

One of our collaborators has [mentioned](https://github.com/okfn/publicbodies/issues/119) Govdirectory, about which I did not know before. Maybe we can find some possible points of collaboration that would benefit both projects.

**Comment by Ainali** [comment]
Thank you for connecting, @augusto-herrmann ! In #91 @loleg mentioned this. I think there are many ways we could collaborate. The biggest difference between our projects is that we are not aiming to host any data ourselves, we will just reuse data in Wikidata.

Have you considered to contribute the data you have and collect there?
